x-pack/plugins/ml/public/embeddables/lens/lens_vis_layer_selection_flyout/layer/compatible_layer.tsx

```diff
@@ -0,0 +1,432 @@
+                {
+                  defaultMessage: 'Additional settings',
+                }
+              )}
+            >
+              <EuiSpacer size="s" />
+              <EuiFormRow
+                label="Bucket span"
+                error={bucketSpanValid}
+                isInvalid={bucketSpanValid !== ''}
+              >
+                <EuiFieldText
+                  data-test-subj={`mlLensLayerBucketSpanInput_${layerIndex}`}
+                  value={bucketSpan}
+                  onChange={(e) => {
+                    setBucketSpan(e.target.value);
+                    setState(STATE.VALIDATING);
+                  }}
+                />
+              </EuiFormRow>
+              <EuiSpacer size="l" />
+              <EuiFormRow>
+                <EuiCheckbox
+                  id="startJob"
+                  data-test-subj={`mlLensLayerStartJobCheckbox_${layerIndex}`}
+                  checked={startJob}
+                  onChange={(e) => setStartJobWrapper(e.target.checked)}
+                  label={i18n.translate(
+                    'xpack.ml.embeddables.lensLayerFlyout.createJobCallout.additionalSettings.start',
+                    {
+                      defaultMessage: 'Start the job after saving',
+                    }
+                  )}
+                />
+              </EuiFormRow>
+
+              <EuiSpacer size="s" />
+              <EuiFormRow>
+                <EuiCheckbox
+                  id="realTime"
+                  disabled={startJob === false}
+                  data-test-subj={`mlLensLayerRealTimeCheckbox_${layerIndex}`}
+                  checked={runInRealTime}
+                  onChange={(e) => setRunInRealTime(e.target.checked)}
+                  label={i18n.translate(
+                    'xpack.ml.embeddables.lensLayerFlyout.createJobCallout.additionalSettings.realTime',
+                    {
+                      defaultMessage: 'Leave the job running for new data',
+                    }
+                  )}
+                />
+              </EuiFormRow>
+              <EuiSpacer size="m" />
+            </EuiAccordion>
+          </EuiForm>
+          <EuiSpacer size="m" />
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <EuiButton
+                disabled={
+                  state === STATE.VALIDATING ||
+                  jobId === '' ||
+                  jobIdValid !== '' ||
+                  bucketSpanValid !== ''
+                }
+                onClick={createADJob.bind(null, layerIndex)}
+                size="s"
+                data-test-subj={`mlLensLayerCreateJobButton_${layerIndex}`}
+              >
+                <FormattedMessage
+                  id="xpack.ml.embeddables.lensLayerFlyout.createJobButton.saving"
+                  defaultMessage="Create job"
+                />
+              </EuiButton>
+            </EuiFlexItem>
+
+            <EuiFlexItem grow={false}>
+              <EuiButtonEmpty
```

Comment:
Could the bucket span and job ID be passed into the wizard if the user has already added / edited these in the flyout, rather than going back to the default `15m` and empty job ID?

Reply:
Not easily.
In a follow up PR I'm going to add the bucket span estimator to this flyout, it needs a bit of refactoring and so is out of scope for this PR.
I can also look at making this change too.